gatherpress.php

```diff
@@ -30,3 +30,40 @@
 GatherPress\Core\Autoloader::register();
 GatherPress\Core\Setup::get_instance();
 GatherPress\BuddyPress\Setup::get_instance();
+
+
+register_activation_hook( __FILE__, 'activate_gatherpress_plugin' );
+/**
+ * Activate GatherPress plugin.
+ *
+ * @return void
+ */
+function activate_gatherpress_plugin() {
+	// $setup = GatherPress\Core\Setup::register_post_types();
+	// $flushing = $setup->register_post_types();
+	if ( ! defined( 'GATHERPRESS_VERSION' ) ) {
+		add_option( 'gatherpress_flush_rewrite_rules_flag', true );
+	}
+}
+
+register_deactivation_hook( __FILE__, 'deactivate_gatherpress_plugin' );
+/**
+ * Activate GatherPress plugin.
+ *
+ * @return void
+ */
+function deactivate_gatherpress_plugin() {
+	flush_rewrite_rules();
+}
+
+add_action( 'init', 'maybe_flush_gatherpress_rewrite_rules', 20 );
```

Comment:
Wouldn't use just random priorities. If you want it to run right after `10` make it `11`. If you want it to run last make it `PHP_INT_MAX`.

Reply:
In the end I was able to remove it altogether